app/models/junior_story.rb

```diff
@@ -119,6 +121,22 @@ def other_sentence
     "#{other}" if other.present?
   end
 
+  def self.to_csv(options = {})
+    csv_string = CSV.generate(options) do |csv|
+      csv << JuniorStory.attribute_names
+      JuniorStory.all.each do |story|
```

Comment:
I was playing around with this method & was getting a bit confused, because you have `JuniorStory.all`, which made me thing that we should just use the junior stories within the `can_publish` scope. But turns out you already define this in the controller - so it was working, but was a bit confusing. 

here's my suggestion:

```
def self.to_csv(options = {})
    csv_string = CSV.generate(options) do |csv|
      csv << attribute_names
      find_each do |story|
        csv << story.attributes.values
      end
    end
  end
```

since you use `self`, you don't need to use the class name anywhere, and the scope is already defined from the controller, so it's a bit clearer. Also, using [`find_each`](http://api.rubyonrails.org/classes/ActiveRecord/Batches.html#method-i-find_each) will cover us in the future for when we have BILLIONS of users and our csv's will just get bigger and bigger :)